theme.json

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
Please add `custom` variables for demonstration, utilize those CSS variables at some place. You can refer [Twenty TwentyTwo](https://github.com/WordPress/twentytwentytwo/blob/trunk/theme.json#L1390) for this.